### DISCUSSION TITLE
Migrating for FlashBuilder

### DISCUSSION THREAD

**Discussion by markdemich** [BowlerHatLLC/vscode-as3mxml#670] [Q&A] [answered]
I have a large workspace in FlashBuilder with several Flex LIbrary projects that are then used in an Air application also included in this workspace.  In Flash Builder I can choose to "build" the air application and it will recompile the projects if they changed since the last compile. Is there a way to do that with this extension?

**Comment by joshtynjala** [comment, marked as answer]
You can use the `dependsOn` field in _.vscode/tasks.json_ to build libraries before the main application.

https://code.visualstudio.com/docs/editor/tasks#_compound-tasks

It will recompile the libraries every time, though, even if nothing changed.

EDIT: I created a Gist that shows how to do it

https://gist.github.com/joshtynjala/7652881ff6b2dc53fc25f2c74d38c2c5

**Reply by markdemich** [reply]
Thanks. I will give that a try.
